datalad/interface/ls.py

```diff
@@ -297,6 +386,75 @@ def _ls_dataset(loc, fast=False, recursive=False, all=False):
         ds_str = format_ds_model(formatter, dsm, full_fmt, format_exc=path_fmt + u"  {msg!R}")
         print(ds_str)
 
+
+def JsonFormatter(path, repo, _type, size, date):
+    pretty_size = humanize.naturalsize(size)
+    pretty_date = time.strftime(u"%Y-%m-%d/%H:%M:%S", time.localtime(date))
+    json_fmt = u'{\"path\": \"%s\", \"repo\": \"%s\", \"type\": \"%s\", \"size\": \"%s\", \"date\": \"%s\"}'
+    return json_fmt % (path, repo, _type, pretty_size, pretty_date)
```

Comment:
I don't think you should bother with manual json formatting... create a dict with all fields, use json.dumps (iirc function correctly). git grep json  since we dump json files in a few places